imports/a_package.go

```diff
@@ -0,0 +1,24 @@
+package imports
+
+import (
+	"github.com/cosmos72/gomacro/base/paths"
+	i "github.com/cosmos72/gomacro/imports"
```

Comment:
Any particular reason for this import renam?

Reply:
Technically, it's not required.
I just preferred not to import a package named "imports" from another package named "imports" too.

Reply:
It shouldn't be a concern. There's no conflict since a package never sees its own name, and this file is very small, so there's no confusion.

Reply:
ok, replaced the named import with a plain one.